test/test_sparse.py

```diff
@@ -102,14 +103,14 @@ def test_print(self):
             printed.append("# indices shape: {}".format(indices_shape))
             printed.append("# values shape: {}".format(values_shape))
 
-            indices = torch.arange(indices_shape.numel(), dtype=self.IndexTensor.dtype,
+            indices = torch.arange(indices_shape.numel(), dtype=self.index_tensor(0).dtype,
```

Comment:
`index_dtype`? ;)

Reply:
I thought about this one but didn't seem worth it.